ci/deploy-rpm.sh

```diff
@@ -1,18 +1,21 @@
 #!/bin/bash
 
+TRIVY_VERSION=$(find ../dist/ -type f -name "*64bit.rpm" -printf "%f\n" | head -n1 | sed -nre 's/^[^0-9]*(([0-9]+\.)*[0-9]+).*/\1/p')
+
 function create_rpm_repo () {
         version=$1
         rpm_path=rpm/releases/${version}/x86_64
 
-        RPM_EL=$(find ../dist/ -type f -name "*64bit.rpm" -printf "%f\n" | head -n1 | sed -e "s/_/-/g" -e "s/-Linux/.el$version/" -e "s/-64bit/.x86_64/")
-        echo $RPM_EL
-
         mkdir -p $rpm_path
-        cp ../dist/*64bit.rpm ${rpm_path}/${RPM_EL}
+        cp ../dist/*64bit.rpm ${rpm_path}/
+
+        createrepo_c -u https://github.com/aquasecurity/trivy/releases/download/ --location-prefix="v"$TRIVY_VERSION --update $rpm_path
```

Comment:
Using `--location-prefix` is better? It is not used in the original version.
https://github.com/trivy-repo-dev/trivy-repo-dev.github.io/blob/5b004d9c8c2fe060f5b70a3b54f348e8e623eaed/deploy-rpm.sh#L10

Reply:
the main branch has `--location-prefix`: https://github.com/trivy-repo-dev/trivy-repo-dev.github.io/blob/b745d6030163c542ffc742760909ca204e67f291/deploy-rpm.sh#L13

but i think it doesn't matter. we can add the version to `-u` flag